src/App.tsx

```diff
@@ -53,7 +55,7 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector value={author} onChange={onSetAuthor} />
```

Comment:
I think you don't need to pass props, you can get state from redux inside the component